src/journal-remote/journal-remote.c

```diff
@@ -144,6 +144,7 @@ static int spawn_curl(const char* url) {
                                 "-HAccept: application/vnd.fdo.journal",
                                 "--silent",
                                 "--show-error",
+                                "--no-buffer",
```

Comment:
You are saying in your commit log "In the case of the --follow option", but this changes it for all occasions. Is that on purpose or should that parameter be made conditional?